src/api/render-node.ts

```diff
@@ -137,6 +137,20 @@ function getMaticReader(snapshot: StateSnapshot, config: ServiceConfiguration) {
   const imageTag = parseImageTag(version);
   if (!imageTag) return undefined;
 
+  const maticConfig: {[index: string]:any} = {
+    ...config.ExternalLaunchConfig,
+    BootstrapMode: false,
```

Comment:
I think this is a bug which will cause management-service to wakeup in bootstrap mode